lnbits/templates/base.html

```diff
@@ -258,6 +258,62 @@
         { value: 'fi', label: 'Suomi', display: '🇫🇮 FI' }
       ]
     </script>
+    <script>
+      const currentUrl = window.location.href
+      let url = new URL(currentUrl)
+      let params = new URLSearchParams(url.search)
+      let q = Quasar.plugins.LocalStorage
+
+      const setColors = () => {
```

Comment:
i think this should be on `created()` in base.js not in the html, i would only put variables there